webcompat/webhooks/helpers.py

```diff
@@ -48,8 +49,22 @@ def set_label(label, issue_number):
     payload = [label]
     api_post('labels', payload, issue_number)
 
+def extract_domain_name(url):
+    # List of major sites where using only the domain results in losing valuable information
+    blackList = r'(\.google\.com|\.live\.com|\.yahoo\.com|go\.com|\.js$)'
```

Comment:
Managed it :) Small question here on the domain blacklist: Do we need to include the \.js?